elevation_mapping:
  ros__parameters:
    input:
      point_cloud: "/zedm/zed_node/point_cloud/cloud_registered"
      pose_covariance: "/zedm/zed_node/pose_with_covariance"
    output:
      raw_map: raw_elevation_map

post_processing:
  ros__parameters:
    input: 
      grid_map: raw_elevation_map
    output:
      grid_map: filtered_elevatoin_map